lib/bamboo/mailer.ex

```diff
@@ -113,7 +123,21 @@ defmodule Bamboo.Mailer do
   end
 
   @doc false
-  def deliver_now(adapter, email, config) do
+  def deliver_now(adapter, email, config, response: true) do
+    email = email |> validate_and_normalize(adapter)
+
+    if email.to == [] && email.cc == [] && email.bcc == [] do
+      debug_unsent(email)
+      email
+    else
+      debug_sent(email, adapter)
+      response = adapter.deliver(email, config)
+      {response, email}
```

Comment:
How did you decide to order the tuple response first then email? At first glance I think I may have expected the order reversed. That's not to suggest that you _should_ reverse the order; I just thought it would be interesting to talk through.

Reply:
Great question! I initially decided to do `{response, email}` since I was thinking of the typical `{:ok, thing}` / `{:error, thing}` pattern and was thinking of `:ok` and `:error` as a response, so the response here would match that and come first. But now that I think about it more, it's different since it's more than just an atom with like a response status, and it comes after the sending of the email, so it probably makes more sense to come second. Plus, since this is an addition to the existing function which just returns an email, it also probably makes sense for the extra information to come after the expected information. I think I will probably reverse it. Thanks!